### PR TITLE
Improvements to issue templates

### PR DESCRIPTION
**Pull request by EPMatt**
Hi @xaviml ,
I added a few details to the different issue templates, in addition to what you had already committed in the past days. I also made some edits to the formatting, let me know what you think about the final result and if there are some changes you'd like to apply.

Regarding the "New device support" template, another great plus could be a doc page describing the process of obtaining action mappings for the integrations ControllerX currently supports. This would provide a guideline for people who want a new device to be added but don't know how to get action mappings. 👍 

P.S. I noticed you made a huge refactor on both the code and the documentation, that's a lot of work! The custom controller feature is just amazing.💪 
Thank you for putting so much effort on ControllerX and bringing this awesome project to the community! 🚀

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -6,27 +6,38 @@ labels: bug
...
 
-_HACS (if installed from there)_:
+## Additional information
 
-_Device_: (All devices involved - including controller, light and media player)
+* Devices involved:
+  * Model: [ eg. IKEA 1744 ] as [ Light | Media Player | Controller ]
+  * ...
+* Integration: Choose from `z2m | deconz | zha`
+* AppDaemon version: [ eg. v5.0.0 ]
```
Comment (xaviml) [inline .github/ISSUE_TEMPLATE/bug_report.md]: The current AppDaemon version is v4.0.1
```suggestion
* AppDaemon version: [ eg. v4.0.1 ]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -6,27 +6,38 @@ labels: bug
...
+* Home Assistant Core version: [ eg. v0.105.4 ]
 
+## AppDaemon app configuration
 
-_Integration_: `z2m | deconz | zha`
-
-_Configuration from `apps.yaml`_:
-```yaml
-Here
 ```
```
Comment (xaviml) [inline .github/ISSUE_TEMPLATE/bug_report.md]: If this is a yaml configuration, we should add the language.
```suggestion
```yaml
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -6,27 +6,38 @@ labels: bug
...
+Insert your appdaemon app configuration here (apps.yaml content).
+Insert only the configuration relative to the faulty controllerx app.
+```
+
+## Logs
 
-_Logs from AppDaemon_:
-```text
-Here
 ```
```
Comment (xaviml) [inline .github/ISSUE_TEMPLATE/bug_report.md]: The same as before:
```suggestion
```text
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 4]
```diff
@@ -8,23 +8,35 @@ assignees: xaviml
...
+* Home Assistant Core version: [ eg. v0.105.4 ]
 
+## AppDaemon app configuration
 
-_Integration_: `z2m | deconz | zha`
-
-_Configuration from `apps.yaml`_:
-```yaml
-Here
+```
```
Comment (xaviml) [inline .github/ISSUE_TEMPLATE/question.md]:
```suggestion
```yaml
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 5]
```diff
@@ -8,23 +8,35 @@ assignees: xaviml
...
+(optional) Insert your appdaemon app configuration here (apps.yaml content).
+Insert only the configuration relative to the controllerx app you're asking the question for.
 ```
 
-_Logs from AppDaemon_:
-```text
-Here
+## Logs
+
+```
```
Comment (xaviml) [inline .github/ISSUE_TEMPLATE/question.md]:
```suggestion
```text
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 6]
```diff
@@ -8,23 +8,35 @@ assignees: xaviml
...
 
-_HACS (if installed from there)_:
+## Additional information
 
-_Device_: (All devices involved - including controller, light and media player)
+* Devices involved:
+  * Model: [ eg. IKEA 1744 ] as [ Light | Media Player | Controller ]
+  * ...
+* Integration: Choose from `z2m | deconz | zha`
+* AppDaemon version: [ eg. v5.0.0 ]
```
Comment (xaviml) [inline .github/ISSUE_TEMPLATE/question.md]:
```suggestion
* AppDaemon version: [ eg. v4.0.1 ]
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (EPMatt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: You're right, my mistake.

**Comment by EPMatt** [comment]
Hi @xaviml,
thanks for the review, I've applied your changes to the PR.

**Comment by xaviml** [comment]
Perfect, thanks @EPMatt

**Pull request merged**